doc/man3/ERR_GET_LIB.pod

```diff
@@ -42,6 +42,13 @@ The library number, reason code, and whether the error
 is fatal, respectively.
 Starting with OpenSSL 3.0.0, the function code is always set to zero.
 
+=head1 NOTES
+
+The application developers SHOULD NOT make their decision on a particular error
+code because they are subject to change even between minor releases of OpenSSL.
```

Comment:
I think this could be said more clearly. How about:

Applications should not make control flow decisions based on specific error codes. Error codes are subject to change at any time (even in patch releases of OpenSSL). A particular error code can only be considered meaningful for control flow decisions if it is explicitly documented as such. New failure codes may still appear at any time.

Reply:
I'm OK with your proposal and can change if you insist or leave as is.

Reply:
I insist on changing the current text (for example I don't think it is clear that "their decision" relates to application logic decisions) - but don't necessary insist on my particular text if someone has a better version.

Reply:
> I insist on changing the current text (for example I don't think it is clear that "their decision" relates to application logic decisions) - but don't necessary insist on my particular text if someone has a better version.

I agree @mattcaswell. It is generally not considered portable to depend on any error code being preserved across platforms. From direct personal experience, even with POSIX, some operations return ENOENT on some platforms at EPERM on others (and sometimes even between OS releases). Different interpretations by underlying system error codes can trigger incompatible responses in higher-level applications and middleware. Application developers should always test there error value assumptions on every platform they support.

Reply:
Pushed, thanks!